cmd/binapi-generator/main.go

```diff
@@ -18,53 +18,64 @@ import (
 	"flag"
 	"fmt"
 	"os"
-	"path/filepath"
-	"strings"
-	"unicode"
 
 	"github.com/sirupsen/logrus"
 
 	"go.fd.io/govpp/binapigen"
 	"go.fd.io/govpp/binapigen/vppapi"
 	"go.fd.io/govpp/version"
+	_ "go.fd.io/govpp/binapigen/plugins"
 )
 
 func init() {
 	flag.Usage = func() {
-		fmt.Fprintf(os.Stderr, "USAGE\n")
-		fmt.Fprintf(os.Stderr, "  Parse API_FILES and generate Go bindings\n")
-		fmt.Fprintf(os.Stderr, "  Provide API_FILES by file name, or with full path including extension.\n")
-		fmt.Fprintf(os.Stderr, "  %s [OPTION] API_FILES\n\n", os.Args[0])
+		fmt.Fprintf(os.Stderr, "GoVPP api bindings generation CLI\n")
+		fmt.Fprintf(os.Stderr, "---------------------------------\n\n")
+		fmt.Fprintf(os.Stderr, "This CLI generates VPP api Go bindings based on .api files or a VPP repository\n\n")
+		fmt.Fprintf(os.Stderr, "Usage: ./binapigen [OPTION]\n")
```

Comment:
`./binapigen` will not be a correct usage most of the time. It will only be correct if the name of the binary is `binapigen` and it's being invoked from the current directory using relative path.

The zeroth argument of the command contains the exact invocation of the program by the user, since it might be anything really. The [`flag` package also uses the `arg[0]`](https://go.dev/src/flag/flag.go#L673) to print the usage. The binary might have different name or be invoked from absolute path or just by running it from the PATH where the `./` would not be present at all.